src/Loop.php

```diff
@@ -59,155 +59,162 @@ public static function stop()
     /**
      * Defer the execution of a callback.
      *
-     * @param callable $callback The callback to defer.
+     * @param callable(mixed $data, string $watcherIdentifier) $callback The callback to defer.
```

Comment:
I included them for consistency and initially thought a watcher could be enabled again, but I see your point. It could be useful in the callback for various reasons, so does it hurt to include it?

Here is a case where object watchers could be preferred to watcher IDs, because a delay or defer watcher could be re-enabled in the loop even after being executed, whereas with a string identifier the watcher becomes invalid.